lib/hub/commands.rb

```diff
@@ -347,12 +347,6 @@ def land(args)
       end
     end
 
-    # $ hub e-note
-    # $ hub e-note "My humble contribution"
-    # $ hub e-note -i 92
-    # $ hub e-note https://github.com/rtomayko/tilt/issues/92
-    alias_method :e_note, :pull_request
```

Comment:
it was an employee joke after Fox News tried to explain Pull Requests as "e-notes": https://twitter.com/huth/status/378306063520374784

since no one actually uses it, it didn't seem worth maintaining in our fork :)